### PR TITLE
Added documentation about assigning user roles

### PR DESCRIPTION
**Pull request by Aminehassou**
## What this PR does
Adds some extra information to `user_roles.md` about the /setting page that allows for easy role assignment.

### CODE REVIEW

File: docs/user_roles.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ permissions, or use another account and select the 'instructor' option from the
 onboarding process (http://localhost:3000/onboarding). Either way, you'll be able
 to create new courses/programs after that from the home page.
 
+## Assigning special roles
+
+To assign users to special roles. Go to the `/settings` page in the dashboard (http://localhost:3000/settings). This will allow you to give users admin privileges and other special user permissions such as classroom program manager, technical help staff, Wikipedia expert and many others. Note that your account needs to have admin permissions to access the page (refer to [Admin permissions](#admin-permissions) for more information)
```
Comment (ragesoss) [inline docs/user_roles.md:44]: Adding special users actually requires `super_admin` permissions (`permissions: 3`), so this should probably add a note about that in the admin section as well.

Comment (Aminehassou) [reply docs/user_roles.md]: Ok, done

File: docs/user_roles.md [hunk 2]
```diff
@@ -32,13 +32,21 @@ You can do so from the rails console:
 - $ `rails c`
 - > `User.find_by(username: '<your username>').update_attribute(:permissions, 1)`
 
+If you need a higher level of permission above admin. Consider giving your main account super admin permissions instead:
```
Comment (ragesoss) [inline docs/user_roles.md:35]: Grammar fix: "If you need a higher level of permission above admin, consider"

File: docs/user_roles.md [hunk 3]
```diff
@@ -32,13 +32,21 @@ You can do so from the rails console:
...
 ## Instructor role
 
 To use the dashboard as an instructor, you can either use an account with admin
 permissions, or use another account and select the 'instructor' option from the
 onboarding process (http://localhost:3000/onboarding). Either way, you'll be able
 to create new courses/programs after that from the home page.
 
+## Assigning special roles
+
+To assign users to special roles. Go to the `/settings` page in the dashboard (http://localhost:3000/settings). This will allow you to give users admin privileges and other special user permissions such as classroom program manager, technical help staff, Wikipedia expert and many others. Note that your account needs to have super admin permissions to access the page (refer to [Admin permissions](#admin-permissions) for more information)
```
Comment (ragesoss) [inline docs/user_roles.md:48]: "To assign users to special roles, go"

Also, add a period at the end.

File: docs/user_roles.md (hunk 2 above)
Comment (Aminehassou) [reply docs/user_roles.md]: Ok, done

File: docs/user_roles.md (hunk 3 above)
Comment (Aminehassou) [reply docs/user_roles.md]: Ok, done

**Pull request merged**